src/core_plugins/markdown_vis/public/_markdown_vis.scss

```diff
@@ -1,5 +1,5 @@
-.markdown-vis {
-  padding: 1em;
+.mkdVis {
+  padding: $euiSize;
```

Comment:
This introduces more padding to the Markdown Vis. I would suggest using a smaller size, especially since when placed on a dashboard, we already add padding via the dashboard. I would rather use `$euiSizeM` here.